gatsby-node.esm.js

```diff
@@ -62,6 +82,30 @@ exports.createPages = async ({ graphql, actions: { createPage } }) => {
       throw result.errors
     }
 
+    result.data.galleryFiftyTwoEntries.edges.forEach((edge, index) => {
+      createPage({
+        path: createFiftyTwoSlug(edge.node.artist.name),
+        component: FiftyTwoDetailPage,
+        context: {
+          id: edge.node.id,
+          prev:
+            index > 0
+              ? createFiftyTwoSlug(
+                  result.data.galleryFiftyTwoEntries.edges[index - 1].node
+                    .artist.name
+                )
+              : null,
+          next:
+            index < result.data.galleryFiftyTwoEntries.edges.length
+              ? createFiftyTwoSlug(
+                  result.data.galleryFiftyTwoEntries.edges[index + 1].node
+                    .artist.name
+                )
+              : null,
```

Comment:
This should be circular:

- when user gets to first entry, clicking previous should take them to last
- when user gets to last entry, clicking next should take them to first